src/test/java/by/teachmeskills/ProductsPageObjectTest.java

```diff
@@ -0,0 +1,31 @@
+package by.teachmeskills;
+
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ProductsPageObjectTest extends BaseTest {
```

Comment:
it is better to name classes based on app functionality. Object is not about app functionality.